Fixing Step #2 for macOS with ARM chips (#21)
Start9Labs/bitcoind-wrapper: docs/integrations/specter/macos.md

:warning: If you have the Tor Browser open, close it and quit the application.

> - For macOS machines prior to December 2020 (pre-ARM chips), in the command line type: ``brew install tor``

Reviewer: ```suggestion
- For macOS, open a terminal and type: ``brew install tor``
```

> - For macOS with ARM chips, in the command line type: ``arch -arm64 brew install tor``

Reviewer: ```suggestion
```

Reviewer: Let's just get rid of this line if `brew install tor` works for you @arjunxkapoor

> - For macOS with ARM chips, in the command line type: ``arch -arm64 brew install tor``

Reviewer: ```suggestion
```

let's just get rid of this line if `brew install tor` works for you @arjunxkapoor
